dali/test/operator_argument.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_TEST_OPERATOR_ARGUMENT_H_
+#define DALI_TEST_OPERATOR_ARGUMENT_H_
+
+#include <string>
+#include <memory>
+#include "dali/pipeline/operators/op_spec.h"
+
+namespace dali {
+namespace testing {
+
+template<typename T>
+struct InputArg {
+  // TODO(mszolucha): For dali::ArgumentInput
```

Comment:
There's no such thing as `dali::ArgumentInput`. Don't use something that looks like an identifier to describe something else.